phonenumber_field/__init__.py

```diff
@@ -1,7 +0,0 @@
-try:
-    from .version import version
-except ImportError:
-    # The version module is written by setuptools_scm.
-    __version__ = None
-else:
-    __version__ = version
```

Comment:
can we make it similar as here https://github.com/ergeon/erg-utils-py/blob/master/ergutils/__init__.py#L71